### ISSUE TITLE
JAWS ignores aria-hidden on links with aria-label in Chromium-based browsers

### ISSUE THREAD

**Issue by JamesCatt** [FreedomScientific/VFO-standards-support#406]
### Summary

Using the up/down arrow keys, navigate to a link that both:

- has an `aria-label`, and
- is set to `aria-hidden="true"` (either directly on the element itself, or on an ancestor element)

### Expected result

JAWS skips past the link without announcing it.

### Actual result

JAWS ignores the hidden state and announces the link.

### Example

https://jamescatt.ca/test/arialabelariahidden/

### Additional Information

Issue is not reproducible with NVDA 2020.1

#### JAWS version and build number

- 2020.2006.12
- 2019.1903.47
 
#### Operating System and version

Windows 10

#### Browser and version:

- Chrome 83.0.4103.116 x64
- Edge 83.0.478.61 x64
- Chromium 78.0.3904.0 (Developer Build) x64

##### Issue does *not* exist with:

- Firefox 78.0.1 x64
- Internet Explorer 11.1550.17134.0

**Comment by JAWS-test** [comment]
It is currently being discussed to ignore aria-hidden at focused elements: https://github.com/w3c/aria/issues/1255.

More test results for this issue:
* in the link overview (INS+F7) the links are not output
* when navigating with the tab key the links are output too
* the links are output with the virtual cursor even if they are not focusable (i.e. have tabindex=-1). I think this is more problematic because your example links are obviously wrong (aria-hidden must not be focusable).

**Comment by JamesCatt** [comment]
> * the links are output with the virtual cursor even if they are not focusable (i.e. have tabindex=-1). I think this is more problematic because your example links are obviously wrong (aria-hidden must not be focusable).

@JAWS-test This is intentional. The scenario where I discovered this was when a modal dialog was open on a page—the underlying page content was set to `aria-hidden`, and javascript was used to prevent the user from focusing any elements outside the dialog.

I was thinking about adding a test case with `tabindex="-1"`, will do so.

**Comment by JAWS-test** [comment]
> The scenario where I discovered this was when a modal dialog was open on a page ...

I agree that this is a valid use case.

**Comment by aleventhal** [comment]
See also https://bugs.chromium.org/p/chromium/issues/detail?id=1105615
Chrome exposes focusable aria-hidden content with STATE_SYSTEM_INVISIBLE, giving the screen reader a chance to announce the content if the user does manage to tab to it.
